### PR TITLE
Added utility module

### PR DESCRIPTION
**Pull request by shahpratham**
This PR adds:
- utility module
- uses convert2relative() from `yolo.py`
- removed hard-coded values of image and model  height, width

### CODE REVIEW

**Review by amanchhaparia — CHANGES_REQUESTED** [review]
@shahpratham, Instead of converting the bbox multiple times just convert the bbox in the required form where it is generated.  Also this branch have some merge conflicts kindly resolve them too.

File: Trackers/iou_tracker/iou.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import numpy as np
 from ..track import Track
 from scipy.optimize import linear_sum_assignment as linear_assignment
+from Detectors.YOLO import yolo
```
[the remark this answers is not in the document]
Comment (shahpratham) [reply Trackers/iou_tracker/iou.py]: yes sorry, mb.

File: Trackers/utility/utility.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    Args
+    detections: 
+    image: A ndarray bgr image.
+
+    Returns
+    image: Annotated image
+    """
+    
+    for bbox, id in detections:
+        left, top, right, bottom =yolo.convert2relative(bbox)
```
[the remark this answers is not in the document]
Comment (shahpratham) [reply Trackers/utility/utility.py]: yes, done

**Comment by shahpratham** [comment]
@amanchhaparia please check now

**Review by amanchhaparia — CHANGES_REQUESTED** [review]
Remove unnecessary `print` statements.

**Review by amanchhaparia — APPROVED** [review]
@shahpratham, Thanks for your work!

**Pull request merged**